### PR TITLE
Test omnisci examples on ci

### PR DESCRIPTION
**Pull request by xmnlab**
This PR:

- updates ibis dependencies
- using the new conda constructor 3.2, now constructor.yaml can define dependencies from an environment.yml using `environment_file` key.
- fixes the issue during the installation when try to run conda init (because conda was not defined as dependency for sh package)
- updates setup-miniconda on ci (`conda-incubator/setup-miniconda@v2`)
- skips packaging for pkg
- tests notebooks from omnisci-examples

### CODE REVIEW

File: constructor/post-install.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 export OLD_PATH=$PREFIX
 export PATH=$PREFIX/bin:$PATH
 
-python -m pip install ibis-vega-transform
+# python -m pip install ibis-vega-transform
```
Comment (xmnlab) [inline constructor/post-install.sh:6]: now defined at https://github.com/Quansight/omnisci-datascience-installer/pull/29/files#diff-f14293d04f2cf788e1bf49f96ac5fbf11cbd5b8206d0e80e7cbcc357c386d371R24

File: README.md [hunk 2]
```diff
@@ -85,10 +85,10 @@ OmniSci Data Science package installs OmniSci stack that includes intake/intake-
...
+**If you have no conda installed on your machine**
 IMPORTANT: If you already have conda installed, this will overwrite the conda path in your bash profile! If you really want to run the following instructions, you probably want to create first a backup of your configuration files (for example, `$HOME/.bashrc`, `$HOME/.bash_profile`, `$HOME/.zshrc`, according to your environment). Otherwise, check the instructions in the sections below.
 
-The OmniSci Data Science Installer can be installed using a shell script if you have no conda on your machine. The shell installers for each version of the package are available on the GitHub  [releases page](https://github.com/Quansight/omnisci-datascience-installer/releases/). For each release, you will find both a MacOS package (`pkg`) and a shell (`sh`) installer. 
+The OmniSci Data Science Installer can be installed using a shell script if you have no conda on your machine. The shell installers for each version of the package are available on the GitHub  [releases page](https://github.com/Quansight/omnisci-datascience-installer/releases/). For each release, you will find both a MacOS package (`pkg`) and a shell (`sh`) installer.
```
Comment (kcpevey) [inline README.md:91]: we should remove the pkg details here

Comment (xmnlab) [reply README.md:91]: sounds good. I will keep the instructions for `pkg` for building.

File: constructor/environment.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  - pydot
+  - pymapd >=0.24
+  - pyarrow 0.15
+  - python 3.7.*
+  - rbc >=0.4
+  # local depencencies
+  - quansight/label/omnisci::holoviews
+  - quansight/label/omnisci::hvplot
+  - pip:
+    - ibis-vega-transform
```
[the remark this answers is not in the document]
Comment (kcpevey) [reply constructor/environment.yaml:24]: ibis-vega-transform is on cf, why the pip install?

[the remark this answers is not in the document]
Comment (xmnlab) [reply constructor/environment.yaml]: it ibis-vega-transform on cf is not updated. also I tried it in the past and the packaging was not working ... I can take a look into that ... but I think that would be good for the next release.

**Comment by xmnlab** [comment]
@kcpevey could you review it again pls? thanks!

**Comment by kcpevey** [comment]
@xmnlab Did you confirm that when you create this env that the udf notebooks will also run?

**Comment by xmnlab** [comment]
I test an udf notebook locally in a linux machine (using the env file) and it worked :)

**Review by kcpevey — APPROVED** [review]

**Pull request merged**